CoreFoundation/Error.subproj/CFError.c

```diff
@@ -495,7 +495,7 @@ static CFTypeRef _CFErrorPOSIXCallBack(CFErrorRef err, CFStringRef key) CF_RETUR
     if (!errStr) return NULL;
     if (CFEqual(key, kCFErrorDescriptionKey)) return errStr;	// If all we wanted was the non-localized description, we're done
     
-#if DEPLOYMENT_TARGET_MACOSX || DEPLOYMENT_TARGET_EMBEDDED || DEPLOYMENT_TARGET_WINDOWS
```

Comment:
Hm, why is this concept not portable to Windows?

Reply:
Windows doesn't really have POSIX-y errors.  We can translate errors using `FormatMessage` which will take care of the localization.  However, Rather than trying to implement something for that, it seemed that `_CFErrorPOSIXCallBack` was invoked from one site, `_CFErrorInitializeCallBackTable`, which is used by `CFError.*CallBackBlockForDomain` which is currently not in use in Foundation ... so I thought I would take the lazy way out :-p.